Forcing.py

```diff
@@ -12,8 +11,11 @@ class ForcingNone():
 	def __init__(self):
 		return
 	def initialize(self, Gr, PV, DV, namelist):
-		self.Tbar = np.zeros((Gr.nx,Gr.ny, Gr.nz), dtype=np.double, order='c')
-		return
+                self.Tbar                       = np.zeros((Gr.nx,Gr.ny,Gr.nz), dtype=np.double, order='c')
+                self.Tbar_layer                 = np.zeros((Gr.nx,Gr.ny      ), dtype=np.double, order='c')
```

Comment:
Tbar_layer is a temporary variable and does not to be part of self it could be defined locally in the loop in update